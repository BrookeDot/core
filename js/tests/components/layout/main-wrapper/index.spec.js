import Component from '@/components/layout/main-wrapper/index'

// Mock manually this component because it changes randomly and breaks the snapshot
jest.mock('@/components/layout/main-wrapper/main-content', () => {
  return {
    render: (h) => h("div")
  }
})

describe('component/layout/main-wrapper/index', () => {
  it('renders properly', async done => {
    const wrapper = await shallow(Component)
    wrapper.vm.$nextTick(() => {
      expect(wrapper).toMatchSnapshot()
      done()
    })
  })
})
