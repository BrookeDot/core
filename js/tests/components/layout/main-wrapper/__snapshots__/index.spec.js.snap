// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`component/layout/main-wrapper/index renders properly 1`] = `
<div id="mainWrapper">
  <sidebar-stub></sidebar-stub>
  <div></div>
  <section id="extra" class="showing">
    <div class="tabs">
      <div role="tablist" class="clear"><button aria-selected="true" aria-controls="extraPanelLyrics" id="extraTabLyrics" role="tab">
          Lyrics
        </button> <button aria-controls="extraPanelArtist" id="extraTabArtist" role="tab">
          Artist
        </button> <button aria-controls="extraPanelAlbum" id="extraTabAlbum" role="tab">
          Album
        </button>
        <!---->
      </div>
      <div class="panes">
        <div aria-labelledby="extraTabLyrics" id="extraPanelLyrics" role="tabpanel" tabindex="0">
          <!---->
        </div>
        <div aria-labelledby="extraTabArtist" id="extraPanelArtist" role="tabpanel" tabindex="0" style="display: none;">
          <!---->
        </div>
        <div aria-labelledby="extraTabAlbum" id="extraPanelAlbum" role="tabpanel" tabindex="0" style="display: none;">
          <!---->
        </div>
        <div aria-labelledby="extraTabAlbum" id="extraPanelAlbum" role="tabpanel" tabindex="0" style="display: none;">
          <!---->
        </div>
      </div>
    </div>
  </section>
  <div class="modal-wrapper">
    <!---->
    <!---->
    <!---->
    <!---->
    <!---->
    <!---->
  </div>
</div>
`;
